frontend/components/ClubEditPage/AddressField.tsx

```diff
@@ -0,0 +1,110 @@
+import { LoadScript } from '@react-google-maps/api'
+import { Libraries } from '@react-google-maps/api/dist/utils/make-load-script-url'
+import { ReactElement, useEffect } from 'react'
+import CreatbleSelect from 'react-select/creatable'
+import usePlacesAutocomplete from 'use-places-autocomplete'
+
+import { BORDER, CLUBS_GREY, FOCUS_GRAY } from '../../constants/colors'
+
+const styles = {
+  control: ({ background, ...base }) => {
+    return {
+      ...base,
+      border: `1px solid ${BORDER}`,
+      boxShadow: 'none',
+    }
+  },
+  option: ({ background, ...base }, { isFocused, isSelected }) => {
+    const isEmphasized = isFocused || isSelected
+    return {
+      ...base,
+      background: isEmphasized ? FOCUS_GRAY : background,
+      color: CLUBS_GREY,
+    }
+  },
+}
+
+type AddressProps = {
+  addressValue: string
+  changeAddress: (state: any) => void
```

Comment:
Is there a reason as to why this is `any` instead of `string`? It seems like you would want to make sure that you always return a string at the end.